src/nimblepkg/options.nim

```diff
@@ -40,6 +40,7 @@ type
     of actionInstall, actionPath, actionUninstall, actionDevelop:
       packages*: seq[PkgTuple] # Optional only for actionInstall
                                # and actionDevelop.
+      installOptions*: seq[string]
```

Comment:
`installOptions` isn't too clear a name

Reply:
well, i need some alternative to `compileOptions2`

Reply:
would `installCompileOptions` be better?

Reply:
It sure is crystal clear (though a bit of a mouthful, but that's usually the trade-off).

Reply:
Why not something like `commandLineFlags` or something that suggest they come from the CLI.

Reply:
well, i took whats already implemented for `build` command. should it be changed too or just this instance?

Reply:
I'd prefer that but @dom96 might want to leave it as is and consistent.

Reply:
This needs to have ``passNim`` in the name... rename it to ``passNimFlags``.